src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -661,13 +661,11 @@ impl<'tcx> Tree {
             for (perms_range, perms) in self.rperms.iter_mut_all() {
                 let idx = self.tag_mapping.get(&tag).unwrap();
                 // Only visit initialized permissions
-                if let Some(p) = perms.get(idx) && p.initialized {
```

Comment:
Why is rustfmt linebreaking this? The code isn't too wide...
I guess we need to add more rustfmt::skip. What it does to the `if` below is also terrible.